src/ServiceFabric.Mocks/MissingActorEventArgs.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using Microsoft.ServiceFabric.Actors;
+
+namespace ServiceFabric.Mocks
+{
+          
+
+    public class MisingActorEventArgs : EventArgs
```

Comment:
How do feel about having a get/set property of type Actor and automatically register that if it gets Set by an event handler?  (similar to how [AssemblyResolve ](https://msdn.microsoft.com/en-us/library/ff527268(v=vs.110).aspx)works)?